github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml

```diff
@@ -1092,6 +1092,125 @@ periodics:
     - name: pgp-bot-key
       secret:
         secretName: pgp-bot-key
+- annotations:
+    testgrid-dashboards: kubevirt-periodics
+    testgrid-days-of-results: "60"
+  cluster: prow-workloads
+  cron: 40 4 * * *
+  decorate: true
+  decoration_config:
+    grace_period: 5m0s
+    timeout: 4h0m0s
+  extra_refs:
+  - base_ref: main
+    org: kubevirt
+    repo: kubevirt
+  - base_ref: main
+    org: kubevirt
+    repo: project-infra
+  labels:
+    preset-bazel-cache: "true"
+    preset-bazel-unnested: "true"
+    preset-dind-enabled: "true"
+    preset-docker-mirror-proxy: "true"
+    preset-kubevirtci-quay-credential: "true"
+    preset-shared-images: "true"
+  reporter_config:
+    slack:
+      job_states_to_report: []
+  name: periodic-kubevirt-performance-cluster-density-test
+  max_concurrency: 1
+  spec:
+    containers:
+    - command:
+      - /usr/local/bin/runner.sh
```

Comment:
This is a lot of code, we should move it to a separate file

Reply:
Thanks for the review @fgimenez 

Where do you think should be the best place to keep such a script?

Reply:
IMO kubevirt's `hack` directory is the most appropriate place.

Reply:
Right, there is already a script in hack, we should update it.
Can we have this PR as it is and another PR later for replacing the `hack/perftests.sh`?

Reply:
sure sgtm